src/config.ts

```diff
@@ -8,7 +10,8 @@ export default {
 
   install_sh_location: {
     server: new URL('https://binaries.ydb.tech/local_scripts/install.sh'),
-    cli: new URL('https://storage.yandexcloud.net/yandexcloud-ydb/install.sh')
+    cli: cliUrl,
+    cli_win: cliUrl
```

Comment:
it should be a different URL: https://storage.ydb.tech/yandexcloud-ydb/install.ps1 (ps1 instead of sh)

Reply:
Got it!